src/libs/NetworkConnection.js

```diff
@@ -47,19 +49,31 @@ function setOfflineStatus(isCurrentlyOffline) {
  * internet connectivity or not. This is more reliable than the Pusher
  * `disconnected` event which takes about 10-15 seconds to emit.
  */
+function subscribeToNetInfo() {
+    NetInfo.configure({
+        // By default, for web (including Electron) NetInfo uses `/` for `reachabilityUrl`
+        // When App is served locally or from Electron this would respond with OK even with no internet
+        // Using API url ensures reachability is tested over internet
+        reachabilityUrl: CONFIG.EXPENSIFY.URL_API_ROOT,
+        reachabilityTest: response => Promise.resolve(response.status === 200),
+    });
```

Comment:
> This should be one time thing

Calling `NetInfo.configure` is (re)checking current network state and we want to invoke it to force a recheck here  `registerConnectionCheckCallback` 

I'll add a comment about it, so that someone doesn't decide to move it out

This is the only way I found to force a state recheck, without this there can be up to a minute delay until you receive network state event that you're offline

Reply:
Oh. Thanks for adding the comment.